### DISCUSSION TITLE
Automatically Export Components in `index` file when using `npx shadcn-ui@latest add [component]`

### DISCUSSION THREAD

**Discussion by alexandredev3** [shadcn-ui/ui#4055] [Ideas]
Currently, we use the following command to add components to our project:
```shell
$ npx shadcn-ui@latest add accordion
```
The component is added to the project. However, we still need to manually create and update the `index.ts` file to export the components if we don't want to import multiple components from separate paths. For example:

```tsx
import { Button } from "@/components/ui/button"
import {
  Card,
  CardContent,
  CardDescription,
  CardFooter,
  CardHeader,
  CardTitle,
} from "@/components/ui/card"
import { Input } from "@/components/ui/input"
import { Label } from "@/components/ui/label"
```

It wouldn't be nice if  Shadcn CLI could automatically create and update a`index` file to export the newly added component? This would allow for a more convenient import syntax:

```tsx
import { Button,  Card, Input, Label, ... } from "@/components/ui"
```

Do we already have a way do this? If not, what are your thoughts on a feature like this? Are there any potential downsides?

**Comment by vitalijalbu** [comment]
i'm also interested, maybe creating a index.ts file into **/components** ?

**Reply by alexandredev3** [reply]
Yes, the CLI could check the components path in the `components.json` and create the `index.ts` file accordingly

**Comment by sovetski** [comment]
Any update on this?
